src/dmd/typesem.d

```diff
@@ -3444,6 +3444,7 @@ Expression dotExp(Type mt, Scope* sc, Expression e, Identifier ident, int flag)
                 tf.purity = PURE.const_;
                 tf.isnothrow = true;
                 tf.isnogc = false;
+                cast(void) tf;
```

Comment:
This looks wrong, seems a bug in dscanner?

Reply:
Yes,  allthough its already fixed in master (we use an older version of dscanner)